config.ts

```diff
@@ -72,6 +73,7 @@ const websiteConfig = z.strictObject({
   discord: z.string(),
   instagram: z.string(),
   linkedin: z.string(),
+  hackathon_url: z.string(),
```

Comment:
Can you put this under `meta` like you do in the type. its a little thing but lets keep them in order.